SignalServiceKit/src/Devices/OWSReadReceiptsForSenderMessage.m

```diff
@@ -67,13 +67,9 @@ - (OWSSignalServiceProtosReceiptMessage *)buildReceiptMessage:(NSString *)recipi
 
 #pragma mark - TSYapDatabaseObject overrides
 
-- (void)saveWithTransaction:(YapDatabaseReadWriteTransaction *)transaction
+- (BOOL)shouldBeSaved
 {
-    // override superclass with no-op.
-    //
-    // There's no need to save this message, since it's not displayed to the user.
-    //
-    // Should we find a need to save this in the future, we need to exclude any non-serializable properties.
+    return NO;
```

Comment:
Would could DRY this up by adding `TSOutgoingSyncMessage` and `TSOutgoingCallMessage` classes, but it doesn't seem worth it.